regression/cbmc/same-object-01/main.c

```diff
@@ -0,0 +1,11 @@
+#include <assert.h>
+#include <stdlib.h>
+
+void main()
+{
+  char *p = malloc(1);
+  assert(__CPROVER_POINTER_OBJECT(p) == 2);
+
+  assert(
```

Comment:
Probably worth asserting that it _isn't_ the same object as something else?

Reply:
Added